src/structures/Guild.js

```diff
@@ -699,21 +692,6 @@ class Guild extends BaseGuild {
       });
   }
 
-  /**
-   * Fetches available voice regions.
-   * @returns {Promise<Collection<string, VoiceRegion>>}
-   */
-  fetchVoiceRegions() {
```

Comment:
This still returns data in api v9... Should we remove it?

Reply:
AFAIK, v9 isn't technically finalized, I think it should be removed now though so it doesn't sit as deprecated until v14.

Reply:
That doesn't mean we should remove this function at all. We don't even know if it WILL be removed in API v9 (and considering VIP regions exist, my gut says this will remain), the only thing we know is removed is guild-level voice regions. This function should personally stay